ClangSharpPInvokeGenerator/CursorVisitor.cs

```diff
@@ -160,6 +160,11 @@ public CXChildVisitResult VisitFunctionDecl(CXCursor cursor, CXCursor parent, CX
                     return Handle(VisitTypeRef, cursor, parent, data);
                 }
 
+                case CXCursorKind.CXCursor_CompoundStmt:
+                {
+                    return CXChildVisitResult.CXChildVisit_Continue;
```

Comment:
It would probably be nice to allow traversing this eventually, but that goes a bit out of scope as compared to just generating bindings (this crops up for methods who's body exist in the header file).